python/ccf/clients.py

```diff
@@ -355,7 +349,7 @@ def request(self, request, timeout=DEFAULT_REQUEST_TIMEOUT_SEC):
 
 class TlsAdapter(HTTPAdapter):
```

Comment:
I believe this class was only used to tweak the Python requests client to use the `secp256k1` curve. Is it still necessary to have when connection to a `secp256r1` node?

Reply:
Good point. I'm not completely sure, but I'll see if all the tests still pass after I remove it.